README.md

```diff
@@ -320,7 +320,56 @@ for (const conversation of myAppConversations) {
 }
 ```
 
-#### Handle different types of content
+### Send a broadcast message
+
+You can send a broadcast message (1:many message or announcement) with XMTP. The recipient sees the message as a DM from the sending wallet address.
+
+1. Use the bulk query `canMessage` method to identify the wallet addresses that are activated on the XMTP network.
+2. Send the message to all of the activated wallet addresses.
+
+For example:
+
+```js
+const ethers = require('ethers')
+const { Client } = require('@xmtp/xmtp-js')
+
+//Rate limits
+//1k publish request /5 minutes / IP
+//publish request would be Client.create() , send() and newConversation()
+//10k of any request type / 5 mins / IP
```

Comment:
what do these comments mean in the context of this code?

Reply:
Removed the comments - ty!